neo4j/data-cron-loader/scripts/reload-data-from-git.sh

```diff
@@ -9,6 +9,7 @@ DATA_OUTPUT_DIR=/shared-data-to-load
 DATA_TMP_DIR=/temp-shared-data-to-load
 
 # to fix warnings in stderr
+cd "$DATA_TMP_DIR/"
 git config pull.rebase false
```

Comment:
Le cd + `git config pull.rebase false` me fait un peu peur. J'imagine qu'on aura une erreur avec `git config` si le dossier `/temp-shared-data-to-load` n'est pas un dépot git (ce qui est le cas au lancement initial  du conteneur).

Est ce que `git pull --no-rebase` suffit à enlever les erreurs stderr ? Est-ce que tu sais si le comportement de 
`--no-rebase` est équivalent à celui de `pull.rebase false` ?

```suggestion
git -C "$DATA_TMP_DIR/" pull --no-rebase || (rm -rf "$DATA_TMP_DIR/*" && git clone "$GIT_REPO" "$DATA_TMP_DIR/") && \
```

ou alors on décide de configurer globalement, et dans ce cas plus besoin de se soucier de l'existence ou non d'un dépôt git dans `/temp-shared-data-to-load`:

```suggestion
git config --global pull.rebase false
```

Reply:
Je ne connais pas le comportement de --no-rebase donc j'aime bien ta seconde suggestion de le faire au niveau global.